### PR TITLE
added whiteblock config/script

### PR DESCRIPTION
**Pull request by steviezhang**
setting up automated testing on a live network with whiteblock

### CODE REVIEW

**Comment by GregTheGreek** [comment]
Can we get this in, I want to test these edge cases we got this week

**Comment by soc1c** [comment]
+1, let's get this in!

**Comment by steviezhang** [comment]
Hey guys! Sorry, I've been waiting on a fix from Whiteblock for an issue I've been running into in the CI container, I'll get this in ASAP after :)

**Comment by soc1c** [comment]
Are we getting this in this week or is this for the next release?

**Comment by GregTheGreek** [comment]
@n8wb any news on the Parity race condition?

**Comment by n8wb** [comment]
Sorry for the late response, I was traveling. The issue was already fixed, it was just that I needed to update your box. So, launch it again and everything should go smoothly.

File: scripts/whiteblock.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+./wb login $JWT 2265
+
+echo "building gethc node..."
+./wb build -b ethclassic --git-repo https://github.com/etclabscore/go-ethereum.git --git-repo-branch development -n 1 -m 0 -c 0 -y -p0=8570:8545
+
+echo "building parity node..."
+./wb build append -b parity -n 1 -m 0 -c 0 -y -p0=8571:8545
+
+echo "pulling latest testing script from github..."
+git clone https://github.com/ChainSafe/Attalus.git
```
Comment (meowsbits) [inline scripts/whiteblock.sh]: https://github.com/ChainSafe/Attalus

I get a 404 here.

**Review by meowsbits — DISMISSED** [review]

File: scripts/whiteblock.sh (hunk 1 above)
Comment (steviezhang) [reply scripts/whiteblock.sh]: It's currently a private repo, we have a deploy key for the CI

Comment (meowsbits) [reply scripts/whiteblock.sh]: So these are secret tests?

Comment (steviezhang) [reply scripts/whiteblock.sh]: https://github.com/ChainSafe/Anemone

File: .circleci/config.yml [hunk 2]
```diff
@@ -35,6 +35,18 @@ jobs:
           key: go-mod-v4-{{ checksum "go.sum" }}
           paths:
             - "/go/pkg/mod"
+  whiteblock-test:
```
Comment (soc1c) [inline .circleci/config.yml]:
```suggestion
  test-whiteblock-go-1.12:
```

File: .circleci/config.yml [hunk 3]
```diff
@@ -87,10 +100,10 @@ workflows:
             - build-linux-go-1.12
           filters:
             tags:
-              only: /.*/
+              only: /.*/ 
       - release-linux-go-1.12:
           requires:
-            - tests-linux-go-1.12
+            - whiteblock-test
```
Comment (meowsbits) [inline .circleci/config.yml]:
```suggestion
            - tests-linux-go-1.12
            - test-whiteblock-go-1.12
```

Comment (meowsbits) [reply .circleci/config.yml]: Assuming that for a (linux, et al) release we don't want to replace tests with whiteblock, but to augment them?

File: .circleci/config.yml (hunk 2 above)
Comment (meowsbits) [reply .circleci/config.yml]: Agreed, but typo: `s/whiteplock/whiteblock/`

**Review by austinabell — APPROVED** [review]
:D

**Review by GregTheGreek — APPROVED** [review]

**Review by amerameen — APPROVED** [review]
Thanks, Steph!
